tests/integration/incremental_strategies/test_incremental_strategies.py

```diff
@@ -77,6 +81,26 @@ def run_and_test(self):
     def test_delta_strategies_databricks_cluster(self):
         self.run_and_test()
 
+# Uncomment this hudi integration test after the hudi 0.10.0 release to make it work.
```

Comment:
Neat! Out of curiosity, what's the change coming in v0.10 that will make this sail smoothly?

Reply:
Spark SQL DML support has been added to Apache Hudi recently with the 0.9.0 release, but there were a few gaps that got fixed after we released the last version, which is scheduled for the next release in a few weeks.

Most specifically, these [commits](https://github.com/apache/hudi/pulls?q=is%3Apr+is%3Aclosed+author%3Apengzhiwei2018) are the ones that are relevant to making these tests run smoothly.